### PR TITLE
📖  DEVELOPING: Add Pull Request section

### PR DESCRIPTION
**Pull request by aghassemi**

### CODE REVIEW

File: contributing/DEVELOPING.md [hunk 1]
```diff
@@ -84,6 +84,22 @@ In particular, we try to maintain "it might not be perfect but isn't broken"-sup
 
 We also recommend scanning the [spec](../spec/). The non-element part should help understand some of the design aspects.
 
+## Code Reviewes
```
Comment (nainar) [inline contributing/DEVELOPING.md]: s/Code Reviewes/Code Reviews

File: contributing/DEVELOPING.md [hunk 2]
```diff
@@ -84,6 +84,22 @@ In particular, we try to maintain "it might not be perfect but isn't broken"-sup
...
+
+Before any code can be merged into the AMP repository, it has to be reviewed and approved by the [owner and a core committer](https://github.com/ampproject/amphtml/blob/master/contributing/owners-and-committers.md).
+
+- If you are contributing a fix or a new feature to AMP, be sure to follow the tips on [preparing your code for review](https://github.com/ampproject/amphtml/blob/master/CONTRIBUTING.md#contributing-code-for-a-feature-coding--implementation-phase) and follow [instructions](https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-e2e.md#send-a-pull-request-ie-request-a-code-review) on how to submit a pull request.
+
+- Pull requests in AMP are triaged similar to Issues and within 2 days of submitting a PR, someone will be assigned as the reviewer. If you have been working with a core committer or someone else already, @mentioning them in the PR description will be helpful.
+
+- After a reviewer is assigned, expect comments on your code within a few days. Feel free to @mention the reviewer if your PR is high priority or if you don't hear back after 5 days since opening the PR.
```
Comment (mrjoro) [inline contributing/DEVELOPING.md]: Would it be realistic if we reduce this to 2 business days?  I don't think it's unreasonable to ask that reviewers at least acknowledge the PR, even if it's to say "Thanks for the PR, I will get to it in 3 days."?

File: contributing/DEVELOPING.md [hunk 3]
```diff
@@ -84,6 +84,22 @@ In particular, we try to maintain "it might not be perfect but isn't broken"-sup
...
+
+- Turnaround time for a PR to be merged is variable and depends on the size and quality of the code. Most PRs go through multiple cycles of review comments. It can take anywhere between a day for simple fixes to weeks for larger features.
+
+- When you have addressed all the review comments from the reviewer, please be sure to @mention them with a **"All requested changes are done, please take another look"** comment in the PR so they would know PR is ready for the next round.
+
+- If your PR is urgent or not progressing at the pace outlined above, please feel free to ping [#contributing](https://amphtml.slack.com/messages/contributing) channel on Slack.
```
Comment (nainar) [inline contributing/DEVELOPING.md]: feel free to ping the

File: contributing/DEVELOPING.md (hunk 2 above)
Comment (nainar) [reply contributing/DEVELOPING.md]: +1

File: contributing/DEVELOPING.md (hunk 1 above)
Comment (aghassemi) [reply contributing/DEVELOPING.md]: I was just trying to help with GFI stuff Naina ;) fixed

File: contributing/DEVELOPING.md (hunk 2 above)
Comment (aghassemi) [reply contributing/DEVELOPING.md]: it is reasonable, part of that would be educating the team to make sure when a PR is assigned to someone via triaging, they would either do the review or add a comment with expected timeline within a day. Historically we haven't been good at this. After this is merged, I will send a PSA email to the team.

**Comment by aghassemi** [comment]
@nainar @mrjoro All requested changes are done, please take another look. 

:D

**Comment by aghassemi** [comment]
@nainar It is already linked actually: https://github.com/ampproject/amphtml/blob/2b080159308c377a5ada9b68b825931dfe5b96f0/contributing/DEVELOPING.md#code-of-conduct

**Pull request merged**